src/lib/stream/shake_cipher/shake_cipher.cpp

```diff
@@ -72,6 +110,7 @@ void SHAKE_Cipher::key_schedule(const uint8_t key[], size_t length)
    const size_t S_pos = SHA_3::absorb(SHAKE_BITRATE, m_state, 0, key, length);
    SHA_3::finish(SHAKE_BITRATE, m_state, S_pos, 0x1F, 0x80);
    copy_out_le(m_buffer.data(), m_buffer.size(), m_state.data());
+   m_buf_pos = 0;
```

Comment:
I suppose that's the bug that you fixed along the way? Worth back-porting?

Reply:
Yes this needs to be backported, bug found by the new tests